rotkehlchen/db/evmtx.py

```diff
@@ -271,14 +272,18 @@ def get_transaction_hashes_not_decoded(
             self,
             chain_id: Optional[ChainID],
             limit: Optional[int],
+            addresses: Optional[List[ChecksumEvmAddress]],
     ) -> List[EVMTxHash]:
         """Get transaction hashes for the transactions that have not been decoded.
         Optionally by chain id.
         If the limit argument is provided then it is used in the SQL query with
         the default order.
         """
         querystr = 'SELECT A.tx_hash from ' + TRANSACTIONS_MISSING_DECODING_QUERY
-        bindings = []
+        bindings: List[Union[ChecksumEvmAddress, int]] = []
+        if addresses is not None:
+            bindings += addresses + addresses
```

Comment:
Why here to use `addresses + addresses` while in the other place (below) `addresses * 2` :thinking:

Reply:
Is because of the type discrepancy between `addresses * 2` and `addresses + addresses` but I agree that looks confusing. WIll try to use the same in both places